.jshintrc

```diff
@@ -20,5 +20,7 @@
 		"Backbone": false,
 		"jQuery": false,
 		"wp": false
-	}
+	},
+
+    "predef": [ "wpsc_vars" ]
```

Comment:
Not sure, but those all sound like edge cases and worst practices.  We develop according to best practices.  We already [declare JShint globals](https://github.com/wp-e-commerce/WP-e-Commerce/blob/master/wpsc-admin/js/media.js#L1) several places in core for JShint, as does [WordPress core](https://github.com/WordPress/WordPress/blob/master/wp-includes/js/autosave.js#L1)